### ISSUE TITLE
Cannot import jobs in XML

### ISSUE THREAD

**Issue by iwd-sys** [rundeck/rundeck#8372]
**Describe the bug**
Jobs in XML that have the `onfailure` plugin cannot be imported anymore. In JSON they are just fine. Error message is `rundeck.controllers.JobXMLException: onfailure plugin had blank or missing 'configuration' element`. This impacts both the UI and the API.

**My Rundeck detail**

* Rundeck version: 4.13.0
* install type: docker
* OS Name/version: Debian Bullseye
* DB Type/version: postgres

**To Reproduce**
1. Export whatever job you want in XML that has a `onfailure` plugin configured
2. Try to reimport the exported job
3. You have an error: `rundeck.controllers.JobXMLException: onfailure plugin had blank or missing 'configuration' element`

**Expected behavior**
Job is imported successfully

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome
 - Version 113

**Additional context**
In JSON export, the `configuration` section exists in the export:

```json
"onfailure" : {
  "plugin" : {
    "configuration" : { },
    "type" : "SlackNotification"
  }
}
```

But in XML, it expect a config to exist but there are none:

```xml
<onfailure>
  <plugin type='SlackNotification' />
</onfailure>
```
Additionally, the [documentation ](https://docs.rundeck.com/docs/manual/document-format-reference/job-v20.html#job) doesn't mention anything about the `configuration` element.

**Comment by MegaDrive68k** [comment]
Hi @iwd-sys 

I'm able to re-import using XML format (on Rundeck 4.13), this is the job, could you compare the definition?

```xml
<joblist>
  <job>
    <defaultTab>nodes</defaultTab>
    <description></description>
    <executionEnabled>true</executionEnabled>
    <id>1d9b3727-9334-4986-8225-03d43699ac70</id>
    <loglevel>INFO</loglevel>
    <name>HelloWorld</name>
    <nodeFilterEditable>false</nodeFilterEditable>
    <notification>
      <onfailure>
        <plugin type='SlackNotification'>
          <configuration>
            <entry key='slack_channel' value='#test' />
            <entry key='webhook_base_url' value='https://hooks.slack.com/services' />
            <entry key='webhook_token' value=' T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX' />
          </configuration>
        </plugin>
      </onfailure>
    </notification>
    <notifyAvgDurationThreshold />
    <plugins />
    <scheduleEnabled>true</scheduleEnabled>
    <sequence keepgoing='false' strategy='node-first'>
      <command>
        <exec>whoami</exec>
      </command>
    </sequence>
    <uuid>1d9b3727-9334-4986-8225-03d43699ac70</uuid>
  </job>
</joblist>
```

**Comment by iwd-sys** [comment]
I did some tests and it appears that none of my jobs are configured with a `<configuration>` marker.

This probably has to do with the recent upgrade I made on Rundeck (I upgraded from 3.4.8 to 4.13.0) and the way default parameter from Slack plugin were handled. When I create a job from scratch it fills the job with the `<configuration>` marker, and I have no issues.

I will then edit manually my jobs to add the configuration needed. Thanks.
